events/wc_dark_cult_events.txt

```diff
@@ -1050,62 +1096,175 @@ narrative_event = {
+			}
+			religion = old_gods_worship
+		}
+
 		remove_trait = travelling
-		
+
+
 		c_hivezora = {
 			# event_target:target_settler and its people settle here
 			settle_as_target_settler_effect = yes
 		}
-		
+
 		# If you're not independent ruler, gives old titles to liege
 		give_old_titles_to_heir_effect = yes
-		
-		d_crystalvale = {
-			any_direct_de_jure_vassal_title = {
-				# event_target:target_settler and its people settle here
-				settle_as_target_settler_effect = yes
-			}
+
+		c_hiveashi = {
+			# event_target:target_settler and its people settle here
+			settle_as_target_settler_effect = yes
 		}
-		d_scarab_wall = {
-			any_direct_de_jure_vassal_title = {
-				# event_target:target_settler and its people settle here
-				settle_as_target_settler_effect = yes
-			}
+		c_crystalvale = {
+			settle_as_target_settler_effect = yes
+		}
+		c_ishrak = {
+			settle_as_target_settler_effect = yes
+		}
+		c_ashakx = {
+			settle_as_target_settler_effect = yes
+		}
+		c_hiveregal = {
+			settle_as_target_settler_effect = yes
+		}
+		c_scarab_wall = {
+			settle_as_target_settler_effect = yes
 		}
-		
 		c_hivezora = {
 			# Declares war against player
 			target_settler_declare_war_effect = yes
 		}
-		
+
 		clear_settler_flag_effect = yes
-		
-		# Spawns army for start
-		spawn_colonial_troops_effect = yes
+		c_hivezora = { location = { culture = qiraji } }
+		c_hiveashi = { location = { culture = qiraji } }
+		c_crystalvale = { location = { culture = qiraji } }
+		c_ishrak = { location = { culture = qiraji } }
+		c_ashakx = { location = { culture = qiraji } }
+		c_hiveregal = { location = { culture = qiraji } }
+		c_scarab_wall = { location = { culture = qiraji } }
 	}
 
 	# I'll spread its will across Kalimdor!
 	option = {
 		name = EVTOPTA_WCDAC_5040
-
-		add_artifact = hammer_of_the_twilight
 		leave_society = yes
 		set_global_flag = twilights_hammer_cult_active_flag
 		join_society_twilights_hammer_cult_effect = yes
 		hidden_effect = {
 			set_society_grandmaster = yes # Set character as Grand Master
 			add_society_currency_massive_effect = yes # Give the Grand Master a start-up capital to induct members
 		}
+		if = {
+			limit = {
+				potential_shadow_priest_wo_training_trigger = yes
+				character = 52000
+			}
+			hidden_effect = {
+				add_character_modifier = { name = training_self_shadow_priest duration = -1 }
+				recount_experience_effect = yes
+				narrative_event = { id = WCCLS.1300 }
+			}
+		}
+		add_35_void_corruption_effect = yes
+
+		hidden_effect = {
+			# activate_title = { title = d_twilights_hammer status = no }
+			# d_twilights_hammer = { destroy_landed_title = THIS }
+			any_playable_ruler = {
+				limit = {
+					ai = yes
+					tier = COUNT
+				}
+				character_event = { id = MNM.10030 }
+			}
+			narrative_event = { id = WCDAC.5050 }
+			character_event = { id = WCDAC.5045 days = 60 }
+		}
+		clr_global_flag = twilights_hammer_kalimdor
+	}
+}
+# Darkness in Ulduar
+narrative_event = {
+	id = WCDAC.5041
```

Comment:
So, from what I see, the only differences between `WCDAC.5040` and `WCDAC.5041` are text and provinces, so it can be the one event with 2 descriptions and if block for province giving.